libs/gi/localization/assets/locales/en/page_character.json

```diff
@@ -8,6 +8,45 @@
     "theorycraft": "Theorycraft",
     "upopt": "Artifact Upgrader"
   },
+  "buildCardTip": {
```

Comment:
Not really an action to change, just more of an observation:
originally the bulk of character related content was on the character page, hence `page_character`, now, most of that is happening in the team page, hence `page_team`. However, things like builds and loadouts are actually shared between the two pages, since the character page has become a loadout/team manager. I am inclined to keep associated l10n in the same json, rather than splitting them by page, so I wonder if it makes sense to create separate l10n namespaces, such as `build.json` and `loadout.json` for l10ns specific to those concepts.

Reply:
@macerator-yaro I discussed with Tech Lead, I think it would make sense to divide up the l10n into separate files. Can you move the build related terms to `build.json`, and loadout related terms to `loadout.json`?